doc/modules/ROOT/pages/python-runtime/knowledge-graph-embeddings.adoc

```diff
@@ -0,0 +1,283 @@
+= Knowledge graph embeddings
+
+Knowledge Graph Embeddings (KGE) refer to a family of algorithms designed to learn low-dimensional representations of entities and relations within a knowledge graph.
+These embeddings are utilized for tasks including link prediction, entity classification, and entity clustering.
+
+This chapter provides an overview of the available shallow embedding algorithms in the GDS Python runtime, such as `TransE` and `DistMult`.
+
+* `TransE`
+** https://proceedings.neurips.cc/paper_files/paper/2013/file/1cecc7a77928ca8133fa24680a88d2f9-Paper.pdf[Bordes, Antoine, et al. "Translating embeddings for modeling multi-relational data." Advances in neural information processing systems 26 (2013).^]
+* `DistMult`
+** https://arxiv.org/pdf/1412.6575[Yang, Bishan, et al. "Embedding entities and relations for learning and inference in knowledge bases." arXiv preprint arXiv:1412.6575 (2014).^]
+
+A knowledge graph is a directed, multi-relational graph.
+It consists of triples `(head, relation, tail)` where `head` and `tail` are entities and `relation` is the relationship between them.
+
+Shallow models of KGE represented here are _inductive_ algorithms for computing node and relationship embeddings (low-dimensional representations) for knowledge graphs.
+Shallow means that matrix lookups represent the entity and relation encoders.
+The algorithm is inductive, meaning that it can be trained on one graph and then applied to this exact graph or another graph with the same schema.
+
+A KGE model can return the score for the triple which can be used to predict the missing tail entity for a given head and relation.
+The embeddings can be used to perform various tasks, such as link prediction, entity classification, and entity clustering.
+
+* https://arxiv.org/pdf/1706.02216.pdf[William L. Hamilton, Rex Ying, and Jure Leskovec. "Inductive Representation Learning on Large Graphs." 2018.^]
+* https://arxiv.org/pdf/1911.10232.pdf[Amit Pande, Kai Ni and Venkataramani Kini. "SWAG: Item Recommendations using Convolutions on Weighted Graphs." 2019.^]
+
+
+[[algorithms-embeddings-kge-considerations]]
+== Scoring functions
+
+For KGE algorithms, the scoring function is used to compute the score of a triple `(head, relation, tail)`.
+This score is used for computing loss during training and during prediction to rank the tail entities for a given head and relation.
```

Comment:
I think we want to say "relationship type" instead of "relation", so as to adjust the terminology to Neo4j

Reply:
Any comment on this?

Reply:
Thanks for noticing it again, I missed this point, now it's fixed